lms/djangoapps/course_structure_api/v0/urls.py

```diff
@@ -18,7 +18,7 @@
         r'^grading_policies/{}/$'.format(COURSE_ID_PATTERN),
         views.CourseGradingPolicy.as_view(),
         name='grading_policy'
-    ),
+    )
```

Comment:
Trailing commas are a great feature of Python: they mean that new lines added after this one won't put this one into the next diff.  See earlier lines in this pull request for examples where you had to add a comma, and now there are two lines in the diff that aren't semantically interesting.